pkg/cache/shares.go

```diff
@@ -160,11 +165,34 @@ func UpdateSharedSecret(share *sharev1alpha1.SharedSecret) {
 	br := share.Spec.SecretRef
 	key := BuildKey(br.Namespace, br.Name)
 	secretsWithShare.Delete(key)
-	AddSharedSecret(share)
+	return
+}
+
+func NamespacesWithSharedConfigMaps() map[string]struct{} {
+	namespacesMap := map[string]struct{}{}
```

Comment:
```suggestion
	namespacesMap := make(map[string]struct{})
```

Reply:
why?

in the past in OCP what I did was explicitly preferred over using `make`

Reply:
That's fine, i thought `make` was the golang gold standard.